### DISCUSSION TITLE
Starting Java Language Server progress.

### DISCUSSION THREAD

**Discussion by davidsoles** [mfussenegger/nvim-jdtls#225] [General]
Hi,
I was wondering, have you seen that the starting progress (%) sometimes (mostly on multi-module apps) show more than 100%? For example, 1100% or 1400%

![Screen Shot 2021-12-30 at 3 12 36 PM](https://user-images.githubusercontent.com/984410/148997592-b558f0b0-9cfc-4154-b25e-061bf57b6f32.png)

What can I do to fix this situation? Thanks...

**Comment by mfussenegger** [comment]
See https://github.com/mfussenegger/nvim-jdtls/issues/218#issuecomment-1011988491
